Sources/PT.PM.Patterns/JavaPatterns.cs

```diff
@@ -448,8 +448,66 @@ public IEnumerable<PatternRoot> CreateJavaPatterns()
                     },
                     Body = new PatternArbitraryDepth
                     {
-                        Pattern = new PatternMethodDeclaration(
-                            Enumerable.Empty<PatternUst>().ToList(), new PatternIdRegexToken(".+"), false)
```

Comment:
In future, all UST and pattern nodes should take IEnumerable as an input parameter in constructors instead of other collections. So, `ToList()` invocation is redundant here.